content/en/synthetics/browser_check.md

```diff
@@ -128,6 +128,15 @@ To use your variables in one of your assertions, hit *Use Variable* and select t
 
 {{< img src="synthetics/browser_check/use_variable_in_assertion.png" alt="Use variable in assertion" responsive="true" style="width:40%;">}}
 
+## External metrics
```

Comment:
I'd put this below a `Data collected`, or a `Metrics` header, just like we do for our integrations page, rather than `External metrics`. I used this term because in the internal doc, it was opposed to the internal metrics that were generated for internal troubleshooting :)